src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -284,5 +259,54 @@ internal static FunctionLoadResponse FunctionLoadRequestHandler(FunctionLoadRequ
 
             return response;
         }
+
+        internal static FunctionEnvironmentReloadResponse EnvironmentReloadRequestHandler(WorkerOptions workerOptions)
+        {
+            var envReloadResponse = new FunctionEnvironmentReloadResponse
+            {
+                Result = new StatusResult { Status = StatusResult.Types.Status.Success },
+                WorkerMetadata = GetWorkerMetadata()
+            };
+            envReloadResponse.Capabilities.Add(GetWorkerCapabilities(workerOptions));
+
+            return envReloadResponse;
+        }
+
+        private static WorkerMetadata GetWorkerMetadata()
+        {
+            var frameworkDescriptionRegex = new Regex(@"^(\D*)+(?!\S)");
+
+            var workerMetadata = new WorkerMetadata
+            {
+                RuntimeName = frameworkDescriptionRegex.Match(RuntimeInformation.FrameworkDescription).Value ?? RuntimeInformation.FrameworkDescription,
+                RuntimeVersion = Environment.Version.ToString(),
+                WorkerVersion = WorkerInformation.Instance.WorkerVersion,
+                WorkerBitness = RuntimeInformation.ProcessArchitecture.ToString()
+            };
+            workerMetadata.CustomProperties.Add("Worker.Grpc.Version", typeof(GrpcWorker).Assembly.GetName().Version?.ToString());
+
+            return workerMetadata;
+        }
+
+        private static IDictionary<string, string> GetWorkerCapabilities(WorkerOptions workerOptions)
```

Comment:
content of this method is existing code. Moved to this method so that it can be called from worker init handler and env reload response handler.